packages/demand-bloc-trip-create/src/Form.ts

```diff
@@ -108,6 +118,17 @@ export class Form {
     this.fields[fieldName].onChange(value)
   }
 
+  validateField(fieldName: string, values: ListOfFields) {
+    this.validationSchema
+      .validateAt(fieldName, values)
+      .then(() => this.fields[fieldName].onError(''))
+      .catch(error => this.fields[fieldName].onError(error.message))
+  }
+
+  validateForm(values: ListOfFields) {
+    for (const field in this.fields) this.validateField(field, values)
```

Comment:
Maybe it's more clear, but it's up to you
`Object.keys(this.fields).forEach(fieldName => this.validateField(fieldName, values)`